plugins/modules/katello_content_view_version.py

```diff
@@ -156,6 +156,7 @@ def promote_content_view_version(module, content_view_version, environments, syn
 
     if promote_to_environment_ids:
         payload = {
+            'id': content_view_version['id'],
```

Comment:
:+1:This i guess is 'the fix' for the referred to problem?

Reply:
Yes.